alica_tests/include/alica_tests/TestWorldModel.h

```diff
@@ -107,6 +116,11 @@ class TestWorldModel : public alica::IAlicaWorldModel
     bool transitionCondition1067314038887345208;
 
     bool switchEntryPoints;
+
+    // Failure handling tests
+    std::atomic<bool> transitionCondition3194919312481305139;
+    std::atomic<bool> transitionCondition1446293122737278544;
+    std::atomic<bool> transitionCondition1023566846009251524;
```

Comment:
Later, these all can be moved to a single unordered_map with key being the transition / condition id. They don't overlap anyways.